keyboards/misterknife/knife66/knife66.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
```

Comment:
Could you add a license header here too?

Reply:
done